todo_app/templates/index.html

```diff
@@ -8,47 +8,67 @@ <h1 class="display-4">To-Do App</h1>
   </div>
  </div>
 	
-    <h2>Create Task</h2>
-    <form action="/create" method="POST">
-        <input type="text" name="title" id="title" value="title">
-        <input type="submit" value="Create">
-    </form>
    
 	<table>
   <tr>
   <div class="row">
     <div class="col-md-12">
-      <th>Items</th>
-	   <th>Status</th>
-	    <th>Item id - Update Status</th>
+     
 	  
 	   </tr>
       <ul class="list-group mb-4">
         <!-- List the todo items here -->
+		  
+		<table>
+		<h3><b>To do</h3></b>                                                                                                       
+		{%for x in my_objects: %}
+		<div>
+				<br>
+				
+				<tr>{{x.name}}<tr>
 		
-		<ul>
-		{% for item in items %}
-		<li><tr>
-		 <form action="/save" method="POST">
-		
-		<td>{{item.title}}  </td> <td> {{item.status}} </td> 
-		<td><form method="POST">
-        <input name="itemId"value="{{item.id}}">
-        <input type="submit"value="Mark as Complete">
-         </form></td>
-       
-    </form>
-		</li>
-		
-		<tr>
-</div>
+		<p><b>Move from doing to done<p></b>
+				<form action="/update" method="post">
+					<input type="hidden" name="id" value="{{x.id}}">
+					<input type="submit" value="Update to done">
+				</form>
+			</div>
 		{% endfor %}
+		<br>
+		<p>Add To do Task<p>
+    <form action="/create" method="POST">
+        <input type="text" name="title" id="title" value="title" required>
+        <input type="submit" value="Create">
+    </form>
+			</table>
+			<table>
+		<br><h3><b> Doing </h3></b>     
+		<tr>{% for listNumberTwo in range(numberTwo) %}<tr>  <br>                   
+	       <tr> {{jsonResponseDoing[listNumberTwo]["name"]}}<tr>
```

Comment:
You can simplify this. `number`, `numberTwo` and `numberThree` are not needed:

```suggestion
		<tr>{% for item in jsonResponseDoing %}<tr>  <br>                   
	       <tr> {{item["name"]}}<tr>
```